alf/examples/ac_target_navigation.gin

```diff
@@ -36,6 +36,9 @@ fc_layer_params=(256,)
 num_state_tiles=800
 num_sentence_tiles=800
 
+mlp_layers.activation_fn=@tf.keras.activations.softsign
+mlp_layers.kernel_initializer=@tf.compat.v1.keras.initializers.glorot_uniform()
```

Comment:
It it possible not to use compat.v1 version?

Reply:
Done.